specification/network/resource-manager/readme.python.md

```diff
@@ -18,9 +18,11 @@ Generate all API versions currently shipped for this package
 
 ```yaml $(python)
 multiapi: true
-default-api-version: "2023-06-01"
+default-api-version: "2023-11-01"
 clear-output-folder: true
 batch:
+  - tag: package-2023-11
+  - tag: package-2023-09
```

Comment:
Since your PR is for 2023-11, I think it is better to add  package-2023-11 only

Reply:
Updated